pyproject.toml

```diff
@@ -40,7 +40,7 @@ selenium = "*"
 webdriver-manager = "*"
 msedge-selenium-tools = "*"
 cssselect = "*"
-lxml = "^4.8.0"
+lxml = "5.3.0"
```

Comment:
This would require all the projects upstream to have this specific version of the `lxml`, should we make it as a requirement or we can provide a range that's suitable?